packages/gatsby/src/services/initialize.ts

```diff
@@ -429,28 +435,30 @@ export async function initialize({
 
       const deleteGlobs = [
         // By default delete all files & subdirectories
-        `${cacheDirectory}/**`,
-        `!${cacheDirectory}/data`,
-        `${cacheDirectory}/data/**`,
-        `!${cacheDirectory}/data/gatsby-core-utils/`,
-        `!${cacheDirectory}/data/gatsby-core-utils/**`,
-        `!${cacheDirectory}/compiled`,
+        `.cache/**`,
```

Comment:
Any reason why you removed `cacheDirectory` usage in the strings?

Reply:
Because I added it to CWD. there is no reason to glob full paths